src/utils/StorybookWrapper.tsx

```diff
@@ -27,6 +27,16 @@ export const mockSubscribe: NewsletterSubscribeCallback = (newsletterId) => {
     });
 };
 
+export const mockRemind: FetchEmail = () => {
```

Comment:
Very minor (feel free to ignore) - what do you think about a slightly different name here, say `mockFetchEmail`? Just to align more closely with what the mock is actually doing (it's not doing any reminding!)